index.js

```diff
@@ -1,9 +1,16 @@
-function moveToMapPosition(referenceMap, mapToMove) {
-  mapToMove.jumpTo({
-    center: referenceMap.getCenter(),
-    zoom: referenceMap.getZoom(),
-    bearing: referenceMap.getBearing(),
-    pitch: referenceMap.getPitch()
+function moveToMapPosition (master, slaves) {
```

Comment:
Could `slaves` be `clones` here? The technical term has been generally changing to either `replica` or `clone` depending on context.